java/apache-solr8/Portfile

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                apache-solr8
+version             8.0.0
+categories          java textproc
+maintainers         nomaintainer
+
+description         An open source enterprise search platform.
+
+long_description    Solr is the popular, blazing fast open source enterprise \
+                    search platform from the Apache Lucene project.
+
+homepage            http://lucene.apache.org/solr/
+platforms           darwin
+supported_archs     noarch
+license             Apache-2
+
+dist_subdir         apache-solr
+distname            solr-${version}
+extract.suffix      .tgz
+
+# curl -O https://www.apache.org/dist/lucene/solr/8.0.0/solr-8.0.0.tgz
+# openssl dgst -rmd160 solr-8.0.0.tgz 
+# RIPEMD160(solr-8.0.0.tgz)= 3512cc7298424ea6221ff86b9f0df9bec631d8b2
+# openssl dgst -sha256 solr-8.0.0.tgz 
+# SHA256(solr-8.0.0.tgz)= 0e6392d3b980ab917c731b054101aafcebceacc0e5063cb1e305aeeaec911d12
```

Comment:
Please remove these commented out lines. Note that the checksums can anyway be generated using the `port -v checksum` command.